docs/workflows/common-workflow-steps/custom-script.md

```diff
@@ -227,87 +227,153 @@ sudo -E printenv
-	sudo apt-get install msmtp-mta
-fi
+cat <<EOF >> ~/.msmtprc
+logfile ~/.msmtp.log
+account $CS_ACCOUNT
+host $CS_HOST
+port $CS_PORT
+from $CS_EMAIL
+user $CS_USERNAME
+password $CS_PASSWORD
+account default: $CS_ACCOUNT
+EOF
+
+# Restrict permissions for the .msmtprc file to avoid security issues
+chmod 600 ~/.msmtprc
+
+# Export mail variables to the shell environment
+shell_rc=~/.$(basename $SHELL)rc
+{
+    echo "export MAIL_SERVER=$CS_HOST"
+    echo "export MAIL_PORT=$CS_PORT"
+    echo "export MAIL_USE_TLS=$CS_USE_TLS"
+    echo "export MAIL_USE_SSL=$CS_USE_SSL"
+    echo "export MAIL_USERNAME=$CS_EMAIL"
+    echo "export MAIL_PASSWORD=$CS_PASSWORD"
+} >> "$shell_rc"
+
+# Send email
+echo "From: $CS_EMAIL_FROM
+To: $CS_EMAIL_TO
+Subject: $CS_EMAIL_SUBJECT
+$CS_EMAIL_BODY" | msmtp --debug --from=$EMAIL_ -t $CS_EMAIL_TO
 
-if [ "$os" == "darwin" ]
-then
-	brew install mailutils
-	brew install msmtp
-	echo "set sendmail=/usr/local/bin/msmtp" | sudo tee -a /etc/mail.rc
-fi
+```
 
-echo "defaults
-auth on
-tls on" >> ~/.msmtprc
-
-if [ "$os" == "linux" ]
-then
-	echo "tls_trust_file /etc/ssl/certs/ca-certificates.crt" >> ~/.msmtprc
-elif [ "$os" == "darwin" ]
-then
-	{ echo -n "tls_fingerprint " &
-	  msmtp --serverinfo --tls --tls-certcheck=off --host=smtp.gmail.com --port=587 \
-	  | egrep -o "([0-9A-Za-z]{2}:){31}[0-9A-Za-z]{2}" ;
-	} >> ~/.msmtprc
-fi
+:::info Script Execution
 
-echo "logfile ~/.msmtp.log
-account gmail
-host smtp.gmail.com
-port 587
-from $EMAIL_
-user $USERNAME_
-password $PASSWORD_
-account default: gmail" >> ~/.msmtprc
-
-echo "export MAIL_SERVER = smtp.gmail.com
-export MAIL_PORT = 587
-export MAIL_USE_TLS = True
-export MAIL_USE_SSL = False
-export MAIL_USERNAME = $EMAIL_
-export MAIL_PASSWORD = $PASSWORD_" >> ~/.$(basename $SHELL)rc
+This script is written in `Bash`. When running with **Custom Script**, you need to set the `Execute With` parameter as `Bash`. For more information, please visit the [**Custom Script Input Variables**](/workflows/common-workflow-steps/custom-script#input-variables) documentation section.
 
-```
+:::
+
+:::tip Reciever Email Address
+
+If you want to send an e-mail to multiple e-mail addresses instead of a single e-mail address. In the `CS_EMAIL_TO` parameter, it will be enough to write all the addresses to send an e-mail separated by commas. For example; `CS_EMAIL_TO=example@email.com,example2@email.com`
+
+:::
 
 :::caution Credentials
 
 When using your own SMTP server credentials for the three variables below, please use Environment Variables. This prevents sensitive information, such as passwords, from being exposed to unauthorized individuals. For more detailed information, please refer to the [**Environment Variables**](/environment-variables/managing-variables) documentation.
 
-- $EMAIL_
-- $USERNAME_
-- $PASSWORD_
+- **$CS_EMAIL**: SMTP Server email address.
+- **$CS_USERNAME**: Sender email address.
+- **$CS_PASSWORD**: Sender email address password.
```

Comment:
For some variables, `$` prefix is not used, but for some of them there is a `$` prefix in writing.

:one: We should have a single style for the same concept. Choose one and stick with that.

:two: Check also other info or tip boxes because I'm not marking them one by one. You can find all of them with short review.